locales/hu_HU/LC_MESSAGES/duckduckgo.po

```diff
@@ -1684,7 +1677,7 @@ msgstr ""
 
 msgctxt "showcase_spread"
 msgid "Help your friends and family join the Duck Side!"
-msgstr "Segíts, hogy barátaid és családod is átálljanak a Duck Oldalra!"
+msgstr "Segíts, hogy barátaid és családod is átálljanak a Kacsa oldalra!"
```

Comment:
@zoli111 We would prefer to keep `Duck` as is, because it's part of our brand. Would `Duck Oldalra` work here?